packages/augur-ui/src/modules/common-elements/labels.tsx

```diff
@@ -730,7 +731,7 @@ export const MovementLabel = (props: MovementLabelProps) => {
 };
 
 export const PillLabel = (props: PillLabelProps) => (
-  <span className={Styles.PillLabel}>{props.label}</span>
+  <span className={classNames(Styles.PillLabel, {[Styles.HideOnMobile]: props.hideOnMobile})}>{props.label}</span>
```

Comment:
now that I'm looking at this, can we destructure those props? so instead of `(props: PillLabelProps)` use `({ label, hideOnMobile }: PillLabelProps)`? I think that it helps make the props more clear at a glance but also lets use not keep doing `props.whatever`, probably should move that way for all the functional components as part of the refactoring efforts.